packages/server-core/src/media/static-resource/static-resource.hooks.ts

```diff
@@ -363,6 +364,38 @@ const addLog = async (context: HookContext<StaticResourceService>, actionType: '
   }
 }
 
+/**
+ * Sync static resource to vector database after create/update
+ */
+const syncToVectorDatabase = async (context: HookContext<StaticResourceService>) => {
```

Comment:
Add a filter here so that we only sync materials and 3d files. Use `FileToAssetType` to determine what is a model/material, and what is not